main.css

```diff
@@ -1186,3 +1186,41 @@ a {
     scale: 1;
   }
 }
+
+
+
+
+/* Nice Gradient for buttons i found :) */
```

Comment:
Wow! It looks cool to a hacker escape room. I really like that the effect goes reverse when moving away from a button. Please check with rest of the team so we work according to the principle that the team shall agree on design. :)